deploy/kubernetes/operator/pkg/controller/constants/constants.go

```diff
@@ -0,0 +1,83 @@
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package constants
+
+const (
+	// ContainerShuffleServerRPCPort indicates rpc port used in shuffle server containers.
+	ContainerShuffleServerRPCPort int32 = 19997
+	// ContainerShuffleServerHTTPPort indicates http port used in shuffle server containers.
+	ContainerShuffleServerHTTPPort int32 = 19996
+	// ContainerCoordinatorRPCPort indicates rpc port used in coordinator containers.
+	ContainerCoordinatorRPCPort int32 = 19997
+	// ContainerCoordinatorHTTPPort indicates http port used in coordinator containers.
+	ContainerCoordinatorHTTPPort int32 = 19996
+
+	// ShuffleServerRPCPortEnv indicates environment name of rpc port used by shuffle servers.
+	ShuffleServerRPCPortEnv = "SERVER_RPC_PORT"
+	// ShuffleServerHTTPPortEnv indicates environment name of http port used by shuffle servers.
+	ShuffleServerHTTPPortEnv = "SERVER_HTTP_PORT"
+	// CoordinatorRPCPortEnv indicates environment name of rpc port used by coordinators.
+	CoordinatorRPCPortEnv = "COORDINATOR_RPC_PORT"
+	// CoordinatorHTTPPortEnv indicates environment name of http port used by coordinators.
+	CoordinatorHTTPPortEnv = "COORDINATOR_HTTP_PORT"
+	// RSSCoordinatorQuorumEnv indicates environment name of rss coordinator quorum used by shuffle servers.
+	RSSCoordinatorQuorumEnv = "RSS_COORDINATOR_QUORUM"
+	// XmxSizeEnv indicates environment name of xmx size used by coordinators or shuffle servers.
+	XmxSizeEnv = "XMX_SIZE"
+	// ServiceNameEnv indicates environment name of service name used by coordinators or shuffle servers.
+	ServiceNameEnv = "SERVICE_NAME"
+	// NodeNameEnv indicates environment name of physical node name used by coordinators or shuffle servers.
+	NodeNameEnv = "NODE_NAME"
+	// RssIPEnv indicates environment name of shuffle servers' ip addresses.
+	RssIPEnv = "RSS_IP"
+
+	// CoordinatorServiceName defines environment variable value of "SERVICE_NAME" used by coordinators.
+	CoordinatorServiceName = "coordinator"
+	// ShuffleServerServiceName defines environment variable value of "SERVICE_NAME" used by shuffle servers.
+	ShuffleServerServiceName = "server"
+
+	// ExcludeNodesFile indicates volume mounting name of exclude nodes file
+	ExcludeNodesFile = "exclude-nodes-file"
+
+	// UpdateStatusError means reason of updating status of rss error
+	UpdateStatusError = "UpdateStatusError"
+
+	// OwnerLabel is the label of configMap's owner.
+	OwnerLabel = "data.tencent.io/owner-label"
```

Comment:
Should we change this from `data.tencent.io` to `org.apache.uniffle`?

Reply:
Sorry, I miss it, I have modified.